ddspipe_yaml/src/cpp/YamlReader_types.cpp

```diff
@@ -428,6 +429,80 @@ void YamlReader::fill(
     }
 }
 
+/************************
+* LOGGING CONFIGURATION *
+************************/
+
+template <>
+DDSPIPE_YAML_DllAPI
+void YamlReader::fill<LogFilter>(
+        LogFilter& object,
+        const Yaml& yml,
+        const YamlReaderVersion version /* version */)
```

Comment:
We usually replace the `version` argument with the `/* version */` comment to indicate that the argument is not used.